tests/test-event-recorder.c

```diff
@@ -275,6 +275,38 @@ test_event_recorder_record_progress_sync_with_floating_key (struct RecorderFixtu
 }
 
 static void
+test_event_recorder_record_progress_with_same_instance_for_key_and_auxiliary_payload (struct RecorderFixture *fixture,
+                                                                                      gconstpointer           unused)
+{
+  GVariant *variant =
+    g_variant_new_string ("Navas clearly deserved the Golden Glove.");
+  g_variant_ref_sink (variant);
+  emtr_event_recorder_record_start (fixture->recorder, MEANINGLESS_EVENT,
+                                    variant, variant);
+  emtr_event_recorder_record_progress (fixture->recorder, MEANINGLESS_EVENT,
+                                       variant, variant);
+  emtr_event_recorder_record_stop (fixture->recorder, MEANINGLESS_EVENT,
+                                   variant, variant);
+  g_variant_unref (variant);
```

Comment:
Is there anything you can assert in these tests? Or is it just to make sure that using the same instance doesn't cause a crash?